tests/SiteSummaryExtensionTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+use BringYourOwnIdeas\SecurityChecker\Tests\Stubs\SiteSummaryAlertStub;
+
+class SiteSummaryExtensionTest extends SapphireTest
+{
+    protected static $fixture_file = 'PackagesWithAlerts.yml';
+
+    protected $requiredExtensions = [
+        SiteSummary::class => [SiteSummaryAlertStub::class]
+    ];
+
+    public function setUp()
+    {
+        parent::setUp();
+        $manifest = new SS_ConfigManifest(BASE_PATH, true);
+        $moduleDoesNotExist = $manifest->moduleExists('bringyourownideas/silverstripe-maintenance');
+        if ($moduleDoesNotExist) {
+            $this->markTestSkipped(
+                'Module bringyourownideas/silverstripe-maintenance is required for this test, but is not present.'
+            );
+        }
+    }
+
+    public function testDisplayColumnsDoesNotIncludePrintingColumns()
+    {
+        $report = Injector::inst()->create('SiteSummary');
+        $this->assertArrayNotHasKey('listSecurityAlertIdentifiers', $report->columns());
```

Comment:
This is a weak test really, can you also check that it is included in one of the other methods, then check that it's not in the printing columns?